device/config/constants.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+const (
+	// Default Configurator constants that are describe a specific configuration option
+	Locale    = "Locale"
+	Keymap    = "Keymap"
+	Wifi      = "Wifi"
+	Interface = "Interface"
+	DNS       = "DNS"
+	SSH       = "SSH"
+
+	MountDir = "/tmp/isaax-sd/"
+
+	Language   = "LANGUAGE=%s\n"
+	LocaleAll  = "LC_ALL=%s\n"
+	LocaleLang = "LANG=%s\n"
+
+	DefaultLocale = "en_US.UTF-8"
+
+	ISAAX_CONF_DIR = "/etc/"
```

Comment:
__[golint]__
* [golint] don't use ALL_CAPS in Go names; use CamelCase

<!-- Comment by [SideCI](https://sideci.com) -->